streams/src/test/java/org/apache/kafka/streams/tests/SmokeTestDriver.java

```diff
@@ -232,7 +236,7 @@ public static void verify(String kafka, Map<String, Set<Integer>> allData, int m
         }
         int retry = 0;
         final long start = System.currentTimeMillis();
-        while (System.currentTimeMillis() - start < TimeUnit.MINUTES.toMillis(3)) {
+        while (System.currentTimeMillis() - start < TimeUnit.MINUTES.toMillis(6)) {
```

Comment:
This is effectively the timeout of the test. With higher replication factor for the internal topics, and with acks to "all" I noticed that the test takes slightly longer to run, so upped this value.